resources/views/emails/user/mobile-verification.blade.php

```diff
@@ -0,0 +1,12 @@
+@component('mail::message')
+    ### We received a request to change your password
+
+    Hi,
```

Comment:
Please add the name of the user here

Reply:
5fd6e5f - Added the name of the user on email templates